client/my-sites/plugins/controller.js

```diff
@@ -255,6 +249,24 @@ const controller = {
 		next();
 	},
 
+	redirectSimpleSitesToPluginBrowser( context, next ) {
+		const site = getSelectedSite( context.store.getState() );
+
+		// Selected site is not a Jetpack site
+		if ( site && ! site.jetpack ) {
+			page.redirect( `/plugins/${ site.slug }` );
+			return;
+		}
+
+		//  None of the other sites are Jetpack sites
+		if ( ! hasJetpackSites( context.store.getState() ) ) {
```

Comment:
Nit: I'd rather write `if ( ! site && ! hasJetpackSites )` here, to make it 100% clear that this check applies only when no site is selected.

Reply:
True, you're right.